x-pack/plugins/siem/public/components/page/network/users_table/translations.ts

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const USERS = i18n.translate('xpack.siem.network.ipDetails.usersTable.usersTitle', {
+  defaultMessage: 'Users',
+});
+
+// Columns
+export const USER_NAME = i18n.translate(
+  'xpack.siem.network.ipDetails.usersTable.columns.userNameTitle',
+  {
+    defaultMessage: 'Name',
+  }
+);
+
+export const USER_ID = i18n.translate(
+  'xpack.siem.network.ipDetails.usersTable.columns.userIdTitle',
+  {
+    defaultMessage: 'Id',
```

Comment:
Should we have the words `Id` and `Group Id` be `ID` and `Group ID` within the column headers?